Tests/ttx/ttx_test.py

```diff
@@ -40,6 +41,13 @@ def temp_font(self, font_path, file_name):
         shutil.copy2(font_path, temppath)
         return temppath
 
+    @staticmethod
+    def read_file(file_path):
```

Comment:
I don't see much the point of this method, you only use it once..
Anyway, keep it, doesn't matter. You still need to pass `encoding="utf-8"` (on Windows it defaults to some cp1252 if I remember well).
And I prefer the `with open(...) as f:` context manager to calling `f.close()`

Reply:
I made it a method because `ttx.py` only has a little over 40% coverage right now, and I expect it will be useful as more tests are added.
I'll make the changes you suggest.